picard/ui/itemviews/columns.py

```diff
@@ -68,15 +69,51 @@ def _sortkey_filesize(obj):
         return 0
 
 
+class IconColumn(ImageColumn):
```

Comment:
I'm not sure about the need of the `ImageColumn` base class and moving `IconColumn` back to itemviews. Isn't this rather generic and could be useful for other cases as well?

Reply:
Not sure yet about this, IconColumn is only used in itemviews for now, and the base ImageColumn is an abstract class for stuff common to such column.

But I open to improvements in this field, navigating by sight right now.